orc8r/cloud/docker/controller/apidocs/v1/swagger.yml

```diff
@@ -6986,6 +6986,16 @@ definitions:
   call_trace_config:
     description: Call Trace spec
     properties:
+      custom_run_options:
+        description: Run options for tcpdump or tshark as it would be typed out in
+          shell
+        example: -ni any -i eth0 -w trace.pcapng
+        type: string
+      custom_run_type:
+        enum:
+        - TCPDUMP
```

Comment:
Are we going to enable user to choose TCPDUMP or TSHARK dynamically on the gateway ? If not, then we can just set the additional arguments as a single param instead of choosing either of the tools.

Reply:
This option has been removed - only tshark will be available